File: fizzbuzz.php
#!/usr/bin/php
#
# FizzBuz in PHP
#

<?php

for ($num = 1; $num <= 100; $num++) {
    $str = $num % 3  != 0 ? "" : "Fizz";

    $str .= $num % 7 != 0 ? "" : "Buzz";

    if ($str !== '') {
      echo $str;
    } else {
      echo $num;
    }
    echo PHP_EOL;
}

File: fizzbuzz.py
#!/usr/bin/python3
#
# FizzBuzz in Python


for num in range(1,101):
  fizz = buzz = ''

  if num % 3 == 0: fizz = 'Fizz' 
  if num % 7 == 0: buzz = 'Buzz' 

  if fizz != '' or buzz != '':
    print(fizz + buzz)
  else:
    print(num)

